docs/csharp/roslyn-sdk/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: The .NET Compiler Platform SDK | Microsoft Docs 
+description: Learn to use the .NET Compiler Platform SDK (also called the Roslyn APIs) to understand .NET code, spot errors, and fix those errors.
+keywords: roslyn, analyzer, code fix
+author: billwagner
+ms.author: wiwagn
+ms.date: 10/10/2017
+ms.topic: conceptual
+ms.prod: .net
+ms.devlang: devlang-csharp
+ms.custom: mvc
+---
+
+# The .NET Compiler Platform SDK
+
+## .NET Compiler Platform SDK Concepts
```

Comment:
usually you have an introduction before jumping to a H2. Maybe move this to after the first paragraph?

also sentence case